AEPServices/Sources/utility/Date+Format.swift

```diff
@@ -12,23 +12,52 @@
 import Foundation
 
 public extension Date {
+
+    /// Returns the number of seconds since the Unix Epoch on 1 January 1970.
+    /// - Returns: the number of seconds since 1 January 1970.
     func getUnixTimeInSeconds() -> Int64 {
         return Int64(timeIntervalSince1970)
     }
 
-    // e.g. - 2020-10-28T15:08:32-06:00
+    /// Returns a string representation of this Date formatted as an ISO 8601 date-time using system local time zone.
+    /// For example, Oct. 28 2020 at 9:08:32.301 am PST is returned as `2020-10-28T09:08:32-08:00`
+    /// - Returns: a string representation of the given Date formatted as an ISO 8601 date-time using system local time zone.
     func getISO8601Date() -> String {
         let formatter = ISO8601DateFormatter()
         formatter.timeZone = TimeZone.current
         formatter.formatOptions.insert(.withInternetDateTime)
         return formatter.string(from: self)
     }
 
-    // e.g. - 2020-10-28T15:08:32-0600
+    /// Returns a string representation of this Date formatted as an ISO 8601 date-time using system local time zone without colons.
+    /// For example, Oct. 28 2020 at 9:08:32.301 am PST is returned as `2020-10-28T09:08:32-0800`
+    /// - Returns: a string representation of the given Date formatted as an ISO 8601 date-time using system local time zone without colons.
     func getISO8601DateNoColon() -> String {
         let formatter = DateFormatter()
         formatter.locale = Locale(identifier: "en_US_POSIX")
         formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssXX"
         return formatter.string(from: self)
     }
+
+    /// Returns a string representation of this Date formatted as an ISO 8601 date-time with fractional seconds, using UTC time zone.
+    /// Use this date format for timestamps send to the Adobe Experience Edge Network.
+    /// For example, Oct. 28 2020 at 9:08:32.301 am PST is returned as `2020-10-28T17:08:32.301Z`
+    /// - Returns: a string representation of the given Date formatted as an ISO 8601 date-time with fractional seconds using UTC time zone.
+    func getISO8601UTCDateInMilliseconds() -> String {
```

Comment:
nit: I feel `getISO8601UTCDateWithMilliseconds` seems more apt for this function.